Final Draught

Cal-7 opened the pull request:
I have combined all the sections, and added introduction/conclusion.
This branch contains exactly all we need; the final draught .qmd file aswell as .pdf , the data, the .gitignore file, and the README file. Therefore, once merged, we will be finished and ready to upload our project.
I think this final draught looks good. Please add any last minute changes before this is merged.

Cal-7 committed:
Adding final draughts to Combining Branch

* FDA final

* Add files via upload

FDA

* Update DA_Group.qmd

* Minor Update Cal

changed path to file to be simpler so everyone can run this. Also fixed a small typo

* Added chi-square and made some modification of the interpretation

* Added a model checking plot

* Update DA_Group.qmd

* Update DA_Group.qmd

* Delete a figure

* Delete DAProject8.csv

Deleted unneeded file

---------

Co-authored-by: Cal-7 <calmcfeat1@gmail.com>
Co-authored-by: nmslsb666cnm <2765145Q@student.gla.ac.uk>

* EDA final

* Completed EDA section and added visuals

* updated the title

* Delete DAProject15.csv

* update EDA qmd

EDA final edits: I removed the table of contents for space, added figure captions and concise interpretations for both visualizations, removed the log transformation plot, and linked figures to the summary table for clearer insights.

* ignored some files to keep branch clean

Moved rendered output to ignored folder to keep branch clean

---------

Co-authored-by: Lanchy-ou <3059059Y@student.gla.ac.uk>
Co-authored-by: nmslsb666cnm <2765145Q@student.gla.ac.uk>
Co-authored-by: Axmedn <154099433+Axmedn@users.noreply.github.com>

Cal-7 committed:
Adding project data

Cal-7 committed:
Combined all Sections

I have also added the introduction and conclusion

Cal-7 committed:
Changed Name of Document

This is the file name they'd like us to use

Cal-7 committed:
Update .gitignore

Ignoring individual files which are not needed for final draught

Cal-7 committed:
Update .gitignore

Cal-7 committed:
Delete eda_analysis.qmd

Cal-7 committed:
Modify .gitignore to let PDF's

I'd like the final PDF to be included, so i needed to modify .gitignore

Cal-7 committed:
Add PDF

Cal-7 committed:
Update .gitignore

Cal-7 committed:
Delete DA_Group.pdf

Cal-7 committed:
Fixed minor bits and added hyperlinks

Axmedn commented:
Looks great and ready to go. We could consider adding a title page with our names, and possibly include references to different sections for easier navigation in the PDF. I’m happy to make those small updates if you’re all okay with it. I'm assuming the extra title page won’t count toward the page limit, but we can confirm just in case.

nmslsb666cnm commented:
I think we can add our names under the title.

Lanchy-ou committed:
Add the label and modified size of each plot

Lanchy-ou committed:
Added the cyberlink on each plot

Cal-7 committed:
Merge branch 'Combining' of https://github.com/nmslsb666cnm/DAS-Group-8 into Combining

Axmedn committed:
Added TOC, resized plots, and updated legend in Figure 3

- Enabled table of contents in PDF
- Resized plots for better layout control
- Added proper legend to Figure 3 (page 5) for clarity

Cal-7 committed:
Merge branch 'main' of https://github.com/nmslsb666cnm/DAS-Group-8 into Combining

Cal-7 committed:
Merge branch 'Combining' of https://github.com/nmslsb666cnm/DAS-Group-8 into Combining

Cal-7 committed:
Removing unwanted files

Cal-7 committed:
White space added

just adding back the vertical space which makes the pdf easier to read

Axmedn committed:
Update README.md

Added project summary, data source, analysis goals, and instructions for generating the report in Quarto.

Axmedn committed:
Update README.md

Axmedn committed:
Update README.md

Axmedn committed:
Update README.md
